src/cryptonote_core/loki_name_system.h

```diff
@@ -157,33 +200,33 @@ struct name_system_db
   // Delete all mappings that are registered on height or newer followed by deleting all owners no longer referenced in the DB
   bool                        prune_db(uint64_t height);
 
-  owner_record                get_owner_by_key      (generic_owner const &owner) const;
-  owner_record                get_owner_by_id       (int64_t owner_id) const;
-  mapping_record              get_mapping           (mapping_type type, std::string const &name_base64_hash) const;
-  std::vector<mapping_record> get_mappings          (std::vector<uint16_t> const &types, std::string const &name_base64_hash) const;
-  std::vector<mapping_record> get_mappings_by_owner (generic_owner const &key) const;
-  std::vector<mapping_record> get_mappings_by_owners(std::vector<generic_owner> const &keys) const;
-  settings_record             get_settings          () const;
+  owner_record                get_owner_by_key      (generic_owner const &owner);
```

Comment:
Getting rid of the const here means the tests need to be adjusted

Reply:
Updated.